crypto/fipsmodule/kem/kem.c

```diff
@@ -306,3 +308,45 @@ int KEM_KEY_set_raw_key(KEM_KEY *key, const uint8_t *in_public,
 
   return 1;
 }
+
+int KEM_KEY_set_raw_keypair_from_seed(KEM_KEY *key, const CBS *seed) {
+  if (key == NULL || seed == NULL || key->kem == NULL) {
+    OPENSSL_PUT_ERROR(EVP, EVP_R_INVALID_PARAMETERS);
```

Comment:
NP: Should we put the error in `CRYPTO` instead of `EVP`? This is currently only called from `EVP` logic, but it could potentially be called from elsewhere.

Reply:
Thanks, switch to use
```
OPENSSL_PUT_ERROR(CRYPTO, ERR_R_SHOULD_NOT_HAVE_BEEN_CALLED);
OPENSSL_PUT_ERROR(CRYPTO, ERR_R_PASSED_NULL_PARAMETER);
OPENSSL_PUT_ERROR(CRYPTO, ERR_R_OVERFLOW);
```
in [69a4ec3](https://github.com/aws/aws-lc/pull/2707/commits/69a4ec3331287f892fc761050034093517e6f4bb)